src/ProjectTemplates/Directory.Build.props

```diff
@@ -6,5 +6,8 @@
     <SkipAnalyzers>true</SkipAnalyzers>
     <ManagePackageVersionsCentrally>false</ManagePackageVersionsCentrally>
     <Api>false</Api>
+    <DisableNETStandardCompatErrors>true</DisableNETStandardCompatErrors>
```

Comment:
Which frameworks do you target that required this?

Reply:
Oh, I see - the same property was used in extensions.  IMO I think it's more readable if you remove that check entirely for template packages.  IIUC the TargetFramework doesn't really have a meaning in template projects.